core/base/Director.cpp

```diff
@@ -1077,13 +1078,6 @@ void Director::cleanupDirector()
     reset();
 
     // cleanup graphics before release glView, otherwise, will cause crash on linux
-    AX_SAFE_RELEASE(_FPSLabel);
```

Comment:
Why remove these release statements?

Reply:
You can see that this function call reset() which already release all of those already, so these release statements is kind of redundant.

However, if you dont want unrelated code change in 1 PR, i totally understand and will do this in a seperate commit / PR.

Reply:
While you're at it, `_notificationNode` is being released and null assigned twice in `reset()`:

```
void Director::reset()
{
...
    if (_notificationNode)
    {
        _notificationNode->onExit();
        _notificationNode->cleanup();
        _notificationNode->release();
    }

    _notificationNode = nullptr;

...
    stopAnimation();    

    AX_SAFE_RELEASE_NULL(_notificationNode);
```

The `AX_SAFE_RELEASE_NULL(_notificationNode);` line can be safely deleted.

Reply:
> You can see that this function call reset() which already release all of those already, so these release statements is kind of redundant.
> 
> However, if you dont want unrelated code change in 1 PR, i totally understand and will do this in a seperate commit / PR.

Yes, revert unrelated changes is better

Reply:
I removed the unrelated changes